src/coordinator/applications.js

```diff
@@ -15,6 +15,29 @@ import { toCamelCase } from '../utils/objects';
  * @property {string} updatedAt ISO 8601 Extended Format date/time string
  */
 
+/**
+ * @typedef {Object} ContxtApplicationGrouping
+ * @property {string} applicationId
+ * @property {ContxtApplicationModule[]} applicationModules
+ * @property {string} id
+ * @property {number} index The position of the grouping within the list of all
+ *   groupings of a the parent application
+ * @property {string} label
+ */
+
+/**
+ * @typedef {Object} ContxtApplicationModule
+ * @property {string} applicationGroupingId
+ * @property {string} [externalLink] A URI pointing to an external application
+ * @property {string} [iconUrl] A URI pointing to an icon/image representing the
+ *   application module
+ * @property {string} id
+ * @property {number} index The position of the module within the list of all
+ *   modules of a the parent application grouping
+ * @property {string} label
+ * @property {string} slug
```

Comment:
`slug` description?
Only because someone that's not familiar with this whole concept wouldn't know what it was